### ISSUE TITLE
[v6] FieldArray ability to remove all

### ISSUE THREAD

**Issue by mauvew** [erikras/redux-form#1095]
Is it possible to provide the ability to remove all elements in a Field array, it can be very useful

**Comment by ooflorent** [comment]
What to you mean by _remove_? Do you want to clear (or reset) the inputs?

**Comment by mauvew** [comment]
yea, it behaves like clear or reset. Let's say we have a form, with one field and one field array. After looking at the api and docs, I didn't find a way to clear all elements in field array. The way I can do currently is to use remove(index) to remove a single element, and iterate the field array

**Comment by ooflorent** [comment]
Correct me if I'm wrong. From what I'm understanding, you have a form with the following fields:

```js
rows[].foo
rows[].bar
```

Given the following values:

```js
{
  rows: [
    {foo: "a1", bar: "b1"},
    {foo: "a2", bar: "b2"},
    {foo: "a3", bar: "b3"},
  ]
}
```

You are looking for a way to _reset_ the values of the specific row. Let's say the second:

```js
{
  rows: [
    {foo: "a1", bar: "b1"},
    {foo: "", bar: ""}, // <-- reset all values of this row
    {foo: "a3", bar: "b3"},
  ]
}
```

Is it what you are trying to do?

**Comment by mauvew** [comment]
thanks for the help. Sorry for the misunderstanding.

What I am trying to do is that, given the following values:
```
{
  fieldOne: 'one',
  rows: [
    {foo: "a1", bar: "b1"},
    {foo: "a2", bar: "b2"},
    {foo: "a3", bar: "b3"},
  ]
}
```

I am looking for a single operation to make the values like the following:

```
{
  fieldOne: 'one',  // <-- not affected
  rows: [] // <-- field array cleared or reset
}
```
Hope it express myself clearly :smile:

**Comment by ooflorent** [comment]
Have you tried using the action creator `change`?

```js
import {change} from "redux-form"
dispatch(change("myform", "rows", []))
```

It don't work in v5 but might in v6.

**Comment by erikras** [comment]
Published as [`6.0.0-alpha-15`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.15).

**Comment by nktssh** [comment]
@erikras maybe, you need update docs site - I see `removeAll` action creator description [in sources](https://github.com/erikras/redux-form/blob/v6/docs/api/ActionCreators.md#arrayremoveallformstring-fieldstring) but not on the [docs site for v6.0.0-rc.3](http://redux-form.com/6.0.0-rc.3/docs/api/ActionCreators.md/)
